Sources/BraintreeCore/Public/BraintreeCore/Braintree.h

```diff
@@ -0,0 +1,4 @@
+#ifndef Braintree_h
+#define Braintree_h
```

Comment:
Do we need this new header?

Reply:
Yes - this was actually a workaround for using Cocoapods in a swift module of a mixed language framework - https://github.com/CocoaPods/CocoaPods/issues/9432#issuecomment-570830515

Happy to implement other solutions, but this was the only one that I was able to get working. I'll add this as an inline comment so the file doesn't seem so out of place.